### ISSUE TITLE
heterocephalus-1.0.0.1 test suite failure from tarball

### ISSUE THREAD

**Issue by DanBurton** [arowM/heterocephalus#4]
As seen on the Stackage build server:

```
expected: sample
          key: <a>,
          key: b,
 but got:
          <interactive>:57:49: error:
              • Exception when trying to run compile-time code:
                  templates/sample.txt: openFile: does not exist (No such file or directory)
                Code: compileTextFile "templates/sample.txt"
              • In the untyped splice: $(compileTextFile "templates/sample.txt")
```

I'm able to reproduce the error on my local machine like so:

```
$ stack unpack heterocephalus-1.0.0.1
$ cd heterocephalus-1.0.0.1/
$ stack init --resolver nightly-2016-11-19
$ stack test
```

This kind of error usually just means that you need to add `templates/sample.txt` to your cabal file under `extra-source-files` so that it gets included in the tarball.

**Comment by arowM** [comment]
@DanBurton Thanks for notifying it.
Could you tell me what should I do for the Stackage error?
Is it enough to release version 1.0.0.2 it does not produce the error and leave v1.0.0.1 as it is?

**Comment by DanBurton** [comment]
Yes, that's fine.

**Comment by arowM** [comment]
I've released v1.0.0.2 that resolves this issue.
